### DISCUSSION TITLE
Turning off CP help

### DISCUSSION THREAD

**Discussion by Gibbanator** [Courseplay/courseplay#7381] [Q&A] [answered]
Is there a way to turn the CP help off in the F1 Help display? If I turn off the CP hub some will go away but not all. Is there I way I can use the CP hub and not have all the CP help options showing in the F1 Help? I have tried looking in game setting and I find nothing.
https://i.gyazo.com/b289149634e1b53d0630e27f865651d9.png

**Comment by Tensuko** [comment, marked as answer]
Advanced Settings -> Page 2 General Settings -> Turn off Keybinds.
